test/TestServerProjects/body-complex/Generated/Models/DateWrapper.Serialization.cs

```diff
@@ -14,12 +14,12 @@ internal void Serialize(Utf8JsonWriter writer)
             if (Field != null)
             {
                 writer.WritePropertyName("field");
-                writer.WriteStringValue(Field.ToString());
+                writer.WriteStringValue(Field.Value.ToString("yyyy-MM-dd"));
```

Comment:
Let's add an extension method for this. Same way we did with query/path/headers.

Reply:
I added an extension method for Utf8JsonWriter called `WriteDateTimeOffsetValue`. This allows you to pass the format specifier to it. Then, that method calls the ToString in TypeFormatters, which is the same logic used for query/path/headers.